### ISSUE TITLE
Adding support for default NetworkPolicy if needed

### ISSUE THREAD

**Issue by farodin91** [utkuozdemir/pv-migrate#158]
**Is your feature request related to a problem? Please describe.**
Our Cluster default denys all traffic, to use pv-migrate, i have to deploy an extra network policy before.

**Describe the solution you'd like**
It would great if pv-migrate would be able to deploy a small network policy which allow traffic between both pods.

**Comment by utkuozdemir** [comment]
Makes sense, thanks for reporting. Will look into it.

**Comment by utkuozdemir** [comment]
The feature is implemented on `v0.12.0` and tested to be working.

The tool does not tap to check if NetworkPolicies are allowed by RBAC or if they are necessary. The user needs to be explicit about them, if they want to use them.

Usage looks like the following:
```bash
pv-migrate migrate .... --helm-set sshd.networkPolicy.enabled=true --helm-set rsync.networkPolicy.enabled=true
```

Feel free to give it a try and give feedback. Thanks.
